gce/composer.lock

```diff
@@ -0,0 +1,3096 @@
+{
```

Comment:
I was wondering if you would notice this :) 

If you deploy an `f1-micro`, there isn't enough memory for composer to install without a lock file. This tutorial uses a `g1-small`, which does deploy without one, so we could potentially drop it.

Reply:
That's really interesting! I wouldn't have guessed the lockfile would affect the memory footprint. Given the context, I still lean removing it - but trust your decision whatever it may be.

Reply:
OK I removed it!